simphony_ui/ui.py

```diff
@@ -98,6 +70,10 @@ class Application(HasStrictTraits):
 
     mlab_model = Instance(MlabSceneModel, ())
 
+    event = Event()
```

Comment:
`calculation_error_event`. Also I _think_ that you can say `Event(Str)` so that the event will constraint what kind of entity you can pass.